pkgs/tools/archivers/arc_unpacker/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchFromGitHub, cmake, makeWrapper, boost, libpng, libjpeg, zlib
+, openssl, libwebp, catch }:
+
+stdenv.mkDerivation rec {
+  pname = "arc_unpacker-unstable";
+  version = "2019-01-28";
+
+  src = fetchFromGitHub {
+    owner = "vn-tools";
+    repo = pname;
+    # the latest release (0.11) doesn't build
```

Comment:
Since we need another iteration anyway, please add a link to the upstream issue in the comment.